packages/data/src/onboarding/resolvers.js

```diff
@@ -89,33 +89,63 @@ function getTasksFromDeprecatedFilter() {
 			alternative: 'TaskLists::add_task()',
 			plugin: '@woocommerce/data',
 		} );
-		return filteredTasks.map( ( task ) => ( {
-			title: task.title,
-			content: task.content,
-			additional_info: task.additionalInfo,
-			time: task.time,
-			level: task.level ? parseInt( task.level, 10 ) : 3,
-			list_id: task.type || 'extended',
-			is_visible: task.visible,
-			id: task.key,
-			is_snoozeable: task.allowRemindMeLater,
-			is_dismissable: task.isDismissable,
-		} ) );
+		return {
+			original: filteredTasks.reduce( ( org, task ) => {
+				return {
+					...org,
+					[ task.key ]: task,
+				};
+			}, {} ),
+			parsed: filteredTasks.map( ( task ) => ( {
```

Comment:
Could this parsing happen server-side?

Reply:
It could yeah, although this seems better of on the client given we are depreciating this on the client.
Also I don't want to be passing any function callback into the `apiFetch` function.

Reply:
The original is also used to merge with the spec coming from the API (line 138). To keep the callback functions around.